examples/real-world/reducers/index.js

```diff
@@ -1,7 +1,7 @@
 import * as ActionTypes from '../actions'
 import merge from 'lodash/merge'
 import paginate from './paginate'
-import { routeReducer } from 'react-router-redux'
+import { reducer as routing } from '../syncHistoryWithStore'
```

Comment:
why not just name the thing 'routing' in syncHistoryWithStore?  I do this exact same thing in my code and my hunch is that other people do as well.  :smile: